packages/datadog-instrumentations/src/express.js

```diff
@@ -57,3 +57,26 @@ addHook({
     })
   })
 })
+
+const processParamsStartCh = channel('datadog:express:process_params:start')
+const wrapProcessParamsMethod = (requestPositionInArguments) => {
+  return (original) => {
+    return function () {
+      if (processParamsStartCh.hasSubscribers) {
+        processParamsStartCh.publish({ req: arguments[requestPositionInArguments] })
+      }
+
+      original.apply(this, arguments)
```

Comment:
I think that currently the original method is not returning nothing. But if in some version they decide to return something to check, with a `return` we will support it.
```suggestion
      return original.apply(this, arguments)
```